src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -38,6 +38,7 @@ $(function () {
 
   function breakText() {
     $(".xref").addClass("text-break");
+    $("td em").addClass("text-break");
```

Comment:
sounds not safe, use a special class to indicate `text-break`, for example, `type-name`?